packages/components/app/styles/components/alert.scss

```diff
@@ -10,6 +10,10 @@
   padding: 1.25rem; // 20px
   display: flex;
   align-content: flex-start;
+
+  code, pre {
+    font-family: var(--token-typography-font-stack-code);
```

Comment:
@didoo @cveigt Just added in this for now, without a background color. 

This overall PR seems ready for review, for merge into https://github.com/hashicorp/design-system/pull/91. Peep the Vercel and LMK if any feedback: https://hds-components-git-amy-base-tweaks-hashicorp.vercel.app/components/alert

Reply:
OK for me to merge in #91 and continue to work branching from there.